package.json

```diff
@@ -17,10 +17,7 @@
     "xkp": "cli.js",
     "xkpasswd": "cli.js"
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/vot/xkpasswd-node.git"
-  },
+  "repository": "https://github.com/vot/xkpasswd-node.git",
```

Comment:
Ah didn't see the last line on last comment. I think this is resolved. Let me make these changes.